src/ngraph/frontend/onnx_import/op/abs.hpp

```diff
@@ -32,7 +33,7 @@ namespace ngraph
             {
                 inline NodeVector abs(const Node& node)
                 {
-                    return {std::make_shared<ngraph::op::Abs>(node.get_ng_inputs().at(0))};
+                    return {std::make_shared<Abs>(node.get_ng_inputs().at(0))};
```

Comment:
Opset1 includes `v0::Abs` so we can use `default_opset.hpp` here.
But we have to decide if `default_opset.hpp` is a good idea or rather explicit include `ngraph/opsets/opset0.hpp`/`ngraph/opsets/opset1.hpp` and use their namespaces explicit before ops.